db/schema.prisma

```diff
@@ -257,3 +259,36 @@ model ContributorPath {
   projectTask     ProjectTasks   @relation(fields: [projectTaskId], references: [id], onDelete: Cascade, onUpdate: Cascade)
   projectStage    ProjectStages  @relation(fields: [projectStageId], references: [id], onDelete: Cascade, onUpdate: Cascade)
 }
+
+model OpportunityAreas {
+  id          String                    @id @default(uuid())
+  name        String
+  description String?
+  sponsor     OpportunityAreasSponsor[]
+  createdBy   String
+  createdAt   DateTime                  @default(now())
+  updatedAt   DateTime                  @default(now())
+  creator     String?
+  project     Projects[]
+  isArchived  Boolean                   @default(false)
+
+  @@index([createdBy], name: "opportunity_areas_createdBy_idx")
+  @@index([isArchived], name: "opportunity_areas_isArchived_idx")
+}
+
+model OpportunityAreasSponsor {
+  id                String            @id @default(uuid())
+  opportunityAreaId String
+  profileId         String
+  status            String?           @default("active")
```

Comment:
What statuses are you thinking? Because for now seems redundant to have both `status` and `active` fields.

Reply:
We could remove the active if the status can be used for it. Then the statuses I was thinking where:
- requested: Someone selects you to be a sponsor
- active: You are a sponsor.
- inactive: You are no longer a sponsor.
I was thinking on the active field as a soft delete in case we wanted to not show it on the UI for any reason, but It may not be necessary